README.md

```diff
@@ -10,6 +10,15 @@ registry.
 
 *see the [cargo docs](https://doc.rust-lang.org/cargo/reference/registries.html#running-a-registry) for details*
 
+## Dependencies
+This project depends mostly on:
+- `rust unstable` (use of feature *XYZ*)
```

Comment:
Let's see if we can't find some best practices for reporting minimum compiler version